pkg/virt-controller/watch/vmi.go

```diff
@@ -234,6 +238,8 @@ type VMIController struct {
 	cdiInformer        cache.SharedIndexInformer
 	cdiConfigInformer  cache.SharedIndexInformer
 	clusterConfig      *virtconfig.ClusterConfig
+	namespaceStore     cache.Store
+	onOpenshift        bool
```

Comment:
Hey @xpivarc ,
Do we really want boolean field called "onOpenshift" in here(or in any other place in our repo) that looks like code smell...